libs/dh/admin/feature-user-roles/src/lib/dh-filter-user-roles-into-table.pipe.ts

```diff
@@ -15,22 +15,22 @@
  * limitations under the License.
  */
 import { Pipe, PipeTransform } from '@angular/core';
+import { DhActorUserRole, DhActorUserRoles } from '@energinet-datahub/dh/admin/shared';
 
 import { WattTableDataSource } from '@energinet-datahub/watt/table';
-import { UserRoleViewDto } from '@energinet-datahub/dh/shared/domain/graphql';
 
 @Pipe({ name: 'filterUserRoles', standalone: true })
 export class FilterUserRolesPipe implements PipeTransform {
-  transform(userRoles: UserRoleViewDto[] | null | undefined, includeAllUserRoles = false) {
-    return (userRoles || []).filter((userRole) => userRole.userActorId || includeAllUserRoles);
+  transform(userRoles: DhActorUserRoles | null | undefined, includeAllUserRoles = false) {
+    return (userRoles || []).filter((userRole) => userRole.assigned || includeAllUserRoles);
```

Comment:
issue: found out why some user roles didn't show up in the UI. It's because of `userRole.assigned`. They'll be filtered out in case `assigned` is `false`.